parser/src/model.ts

```diff
@@ -5,6 +5,12 @@ export interface GitBaseObject {
   hash : string
 }
 
+export interface ParserData {
```

Comment:
Good Idea to move this here. Maybe other statistics should also be moved to this object from `HydratedCommitObject`, such as `minNoCommits` and `maxNoCommits`